providers/implementations/rands/seeding/rand_unix.c

```diff
@@ -33,6 +33,9 @@
 # include <sys/sysctl.h>
 # include <sys/param.h>
 #endif
+#if defined(__FreeBSD__) && __FreeBSD_version >= 1200061
```

Comment:
This seems like it will reintroduce the "bug" fixed by https://github.com/openssl/openssl/commit/1fa90bb3a9089f974e9b07f3e76a964bd2da6976.  However, that original commit looks too narrow to me.  Presumably you don't want any of the OS-specific headers in this section if building for UEFI.  I'm not sure if it would be better to be using other #ifdef's here (e.g. OPENSSL_SYS_LINUX) or if the !OPENSSL_SYS_UEFI should just be a larger #ifndef around this entire section of headers, e.g.:

```
#ifndef OPENSSL_SYS_UEFI
# ifdef __linux
...
# endif
# if defined(__FreeBSD__) || defined(__NetBSD__)
...
# endif
...
# if defined(__DragonFly__)
...
# endif
#endif
```

Reply:
CC @bcran

Reply:
If nobody else objects (nobody has in the last two weeks?) then I'll do that in the next day or so.